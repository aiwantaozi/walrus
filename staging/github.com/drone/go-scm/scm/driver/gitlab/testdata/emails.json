[
    {
        "id": 1,
        "email": "john@company.com",
        "confirmed_at": "2021-03-26T19:07:56.248Z"
    },
    {
        "id": 3,
        "email": "jane@company.com",
        "confirmed_at": null
    }
]