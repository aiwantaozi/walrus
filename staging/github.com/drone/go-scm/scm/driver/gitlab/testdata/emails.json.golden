[
    {
        "Value": "john@company.com",
        "verified": true,
        "primary": false
    },
    {
        "Value": "jane@company.com",
        "verified": false,
        "primary": false
    }
]